### DISCUSSION TITLE
Discord server for Cozodb

### DISCUSSION THREAD

**Discussion by ansarizafar** [cozodb/cozo#148] [General]
Many open source projects are using Discord for discussions and community support nowadays.  @zh217 If you are interested then I can setup and manage a discord server for cozodb.

**Comment by creatorrr** [comment]
Seconded. I’d join it if you start one :)

**Comment by creatorrr** [comment]
I created one coz it seems like there's at least a few people that'd benefit from one. Please feel free to join:
https://discord.gg/DMHUyG2RFA

😄

**Reply by creatorrr** [reply]
@ansarizafar thanks for floating this idea!
